t/45_ods.t

```diff
@@ -28,13 +28,19 @@ my $content;
     #like ($@, qr/too short/);
     }
 
+my $ods_fh;
+{   open $ods_fh, '<', "files/test.ods" or die "files/test.ods: $!\n";
+    }
+
 foreach my $base ( [ "files/test.ods",		"Read/Parse ods file" ],
 		   [ "files/content.xml",	"Read/Parse xml file" ],
 		   [ $content,			"Parse xml data" ],
+		   [ $ods_fh,	"Read/Parse filehandle" ],
 		   ) {
     my ($txt, $msg) = @$base;
     my $sxc;
-    ok ($sxc = ReadData ($txt), $msg);
+    my @options = ref $txt ? ( parser => 'ods'): ();
```

Comment:
I prefer double quotes for strings: `my @options = ref $txt ? (parser => "ods") : ();` or `my @options = ref $txt ? (parser => "ods") : ();`